README.md

```diff
@@ -13,14 +13,14 @@ Also, don't forget to check out [the documentation](https://aka.ms/dotnet-cli-do
 
 Changes in issue triaging
 -------------------------
-As it was explained in the ["Changes to project.json" blog post](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/), we are making significant changes to the CLI tooling by supporting MSBuild as the build engine and `csproj` as the project format. As we are starting that work, we've realized that this also means that many of the shortcomings of the current tooling will be solved by this move. Given this, we've decided to introduce two new labels for issues to indicate that. These labels are: 
+As it was explained in the ["Changes to project.json" blog post](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/), we are making significant changes to the CLI tooling by supporting MSBuild as the build engine and `csproj` as the project format. We've realized that this also means that many of the shortcomings of the current tooling will be solved by this move. Given this, we've decided to introduce two new labels for issues to indicate that. These labels are: 
 
 * `msbuild-mitigated` label - this label indicates that we belive the issue in question will be mitigated via move to MSBuild. 
```

Comment:
You have "label" three times in a row. You only need two of them. This was covered in my earlier feedback.